### PR TITLE
Workshop editing

### PR DESCRIPTION
**Pull request by mpickering**
I edited the first five sections of the workshop to try to make things clearer and add missing parts.

### CODE REVIEW

File: workshop/01-foldDyn.md [hunk 1]
```diff
@@ -171,11 +219,11 @@ We wire this up in a recursive do block so that we can feed the dynamic that we
 
 The the variables that are defined in the rec can have cycles, but are still defined at the same level as the display. It's good to keep the rec blocks as small as is possible so that causality loops (FRP infinite loops) are easier to spot and fix.
 
-Using rec or mdo forces that context to have MonadFix (the abstraction used to tie the knot in the cycle). Doco about the syntax can be found [on the haskell wiki](https://wiki.haskell.org/MonadFix).
+Using rec or mdo forces that context to have `MonadFix` (the abstraction used to tie the knot in the cycle). Documenation about the syntax can be found [on the haskell wiki](https://wiki.haskell.org/MonadFix).
```
Comment (endgame) [inline workshop/01-foldDyn.md:222]: `rec` and `mdo` are keywords too, so please code-quote them.

Comment (mpickering) [reply workshop/01-foldDyn.md:222]: Also typo "documentation".

**Comment by endgame** [comment]
@mpickering I'm not across all the technical details but I think this is an improvement and am prepared to merge. Can you update the last little things we found?

**Comment by benkolera** [comment]
Yeah, lgtm.

**Comment by endgame** [comment]
Looks like the nits aren't going to get picked any time soon but this is still a net improvement.

**Pull request merged**